pkg/inject/config.go

```diff
@@ -150,17 +154,21 @@ func (cfg *Config) BindFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&cfg.EnableXrayTracing, flagEnableXrayTracing, false,
 		"Enable Envoy X-Ray tracing integration and injects xray-daemon as sidecar")
 	fs.Int32Var(&cfg.XrayDaemonPort, flagXrayDaemonPort, 2000,
-		"Datadog Agent tracing port")
+		"X-Ray Agent tracing port")
+	fs.StringVar(&cfg.XraySamplingRate, flagXraySamplingRate, "",
+		"X-Ray tracer sampling rate")
 	fs.StringVar(&cfg.XRayImage, flagXRayImage, "amazon/aws-xray-daemon",
 		"X-Ray daemon container image")
 	fs.BoolVar(&cfg.EnableStatsTags, flagEnableStatsTags, false,
 		"Enable Envoy to tag stats")
 	fs.BoolVar(&cfg.EnableStatsD, flagEnableStatsD, false,
 		"If enabled, Envoy will send DogStatsD metrics to 127.0.0.1:8125")
 	fs.StringVar(&cfg.StatsDAddress, flagStatsDAddress, "127.0.0.1",
-		"Datadog Agent address")
+		"DogStatsD Agent address")
 	fs.Int32Var(&cfg.StatsDPort, flagStatsDPort, 8125,
-		"Datadog Agent tracing port")
+		"DogStatsD Agent tracing port")
+	fs.StringVar(&cfg.StatsDSocketPath, flagStatsDSocketPath, "",
```

Comment:
I notice that there is default value for StatsDAddress. It is better to update description of these parameters to suggest that they will be ignored if StatsDSocketPath is provided.

Reply:
Thanks for the suggestion. I will update the description to include this info.

Reply:
I included this description on helm README.md and the place where we create the env variables.